tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -226,41 +216,42 @@ public void setCacheOnly(boolean cacheOnly) {
      */
     private boolean verboseErrors = false;
 
-    private Calendar calendar = Calendar.getInstance(ZONE_SAMPLE, ULocale.ENGLISH);
+    private final Calendar calendar = Calendar.getInstance(ZONE_SAMPLE, ULocale.ENGLISH);
+
+    private final CLDRFile cldrFile;
 
-    private CLDRFile cldrFile;
+    private final CLDRFile englishFile;
 
-    private CLDRFile englishFile;
-    private BestMinimalPairSamples bestMinimalPairSamples = null;
+    private final BestMinimalPairSamples bestMinimalPairSamples;
 
-    private ExampleCache exCache = new ExampleCache();
+    private final ExampleCache exCache = new ExampleCache();
 
-    private ICUServiceBuilder icuServiceBuilder = new ICUServiceBuilder();
+    private final ICUServiceBuilder icuServiceBuilder = new ICUServiceBuilder();
 
-    private PluralInfo pluralInfo;
+    private final PluralInfo pluralInfo;
```

Comment:
Just a note, this refactoring would be better off in a separate commit or even better a separate PR. Makes it harder to see the actual semantic changes.

Reply:
yes -- I'm trying to get into a habit of doing a preliminary clean-up pr for each ticket that's likely to modify files that need it, but I failed to do that in this case